exercises/practice/paasio/src/test/java/PaasioTest.java

```diff
@@ -0,0 +1,337 @@
+import org.junit.jupiter.api.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.*;
```

Comment:
Please don't use `*` imports, as per other tests ([poker](https://github.com/exercism/java/blob/main/exercises/practice/poker/src/test/java/PokerTest.java) for example).